workflows/spaceranger/spaceranger_count.wdl

```diff
@@ -263,7 +270,7 @@ task run_spaceranger_count {
             call_args.append('--image=~{image}')
         elif len(darkimages) > 0:
             call_args.extend(['--darkimage=' + x for x in darkimages])
-            if has_cyta and '~{dapi_index} != '':
+            if has_cyta and dapi_idx != '':
```

Comment:
This can be
```python
if has_cyta and "~{dapi_idx}" != "":
```

Reply:
Prone to side effects, I will put it like this, but it might randomly fail, as it was before